src/ios/BranchSDK.m

```diff
@@ -152,7 +146,7 @@ - (void)getLatestReferringParams:(CDVInvokedUrlCommand*)command
         pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsDictionary:sessionParams];
     } else {
         NSLog(@"No data found");
-        pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsBool:FALSE];
+        pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:@"No data found"];
```

Comment:
I'd err on leaving this as messageAsBool:FALSE. I had made the same change in Java because (1) Java wouldn't let me return a null, and (2) the most likely use case for someone is to say, in javascript:

```javascript
window.Branch.getLatestReferringParams().then(function(result) { 
  if (result) {
    .. // do logic
  }
});
```

instead of 

```javascript
window.Branch.getLatestReferringParams().then(function(result) { 
  if (result !== "No data found") {
    .. // do logic
  }
});
```

If we leave it as a string, the former code snippet will return a falsy value of true, which is not what we want. Same for the other changes that changed messageAsBool to string.